### PR TITLE
Adding wait for cluster status

### PR DESCRIPTION
**Pull request by iovadia**
Signed-off-by: iovadia <iovadia@redhat.com>

/cc  @YuviGold @nshidlin

### CODE REVIEW

**Review by RazRegev — APPROVED** [review]

File: discovery-infra/test_infra/helper_classes/cluster.py [hunk 1]
```diff
@@ -750,6 +750,14 @@ def wait_for_cluster_to_be_in_finalizing_status(self):
             break_statuses=[consts.ClusterStatus.ERROR]
         )
 
+    def wait_for_cluster_to_be_in_status(self,statuses):
+        utils.wait_till_cluster_is_in_status(
+            client=self.api_client,
+            cluster_id=self.id,
+            statuses=statuses,
+            timeout=consts.CLUSTER_INSTALLATION_TIMEOUT,
```
Comment (RazRegev) [inline discovery-infra/test_infra/helper_classes/cluster.py:758]: there are a few timeouts for multiple statuses, let it by passed as argument with default value

Comment (RazRegev) [reply discovery-infra/test_infra/helper_classes/cluster.py:758]: /hold
unhold after fix

Comment (iovadia) [reply discovery-infra/test_infra/helper_classes/cluster.py]: /unhold

Comment (iovadia) [reply discovery-infra/test_infra/helper_classes/cluster.py]: /cc @lalon4

**Review by RazRegev — APPROVED** [review]

**Pull request merged**